interop-tests/src/test/scala/akka/grpc/scaladsl/LoadBalancingIntegrationSpec.scala

```diff
@@ -55,7 +56,7 @@ class LoadBalancingIntegrationSpec extends AnyWordSpec with Matchers with Before
     }
 
     "re-discover endpoints on failure" in {
-      val service1materializer = akka.stream.ActorMaterializer.create(system)
+      val service1materializer = Materializer(system)
```

Comment:
Is there a relevant difference to `SystemMaterializer(system).createAdditionalSystemMaterializer()` used in `NonBalancingIntegrationSpec`?